components/Post/PageProgress.tsx

```diff
@@ -3,21 +3,30 @@ import { useEffect, useRef } from "react";
 
 const chromeZoomPixelGapBugFix = -0.25;
 
-const PageProgress = () => {
+interface PageProgressProps {
```

Comment:
```suggestion
interface Props {
```

@fordelord just use a generic `Props` name

There's no point to specify it further, as you are already in the `PageProgress.tsx` file, so this name is kind of implied